python/pylibcudf/pylibcudf/join.pyi

```diff
@@ -3,8 +3,8 @@
 
 from enum import IntEnum
 
-from rmm.pylibrmm import Stream
 from rmm.pylibrmm.memory_resource import DeviceMemoryResource
+from rmm.pylibrmm.stream import Stream
```

Comment:
Previously, `from rmm.pylibrmm import Stream` failed to import and `Any` was used. Now this is type checked.